ansible/playbooks/roles/compatibility_layer/tasks/install_packages.yml

```diff
@@ -28,4 +28,6 @@
     group: "{{ username_on_host.stdout }}"
     path: "{{ gentoo_prefix_path }}"
     recurse: true
-  become: false
+  become: true
```

Comment:
Also here we should clarify why `true` (or `false` is needed, this `become` bit is a bit cryptic I think...

Reply:
If the ansible script is run as a normal user, it cannot change ownership of files it doesn't own. That's the purpose of the task ... to change ownership to the user running the task.

Reply:
Added a comment.